testsuite/performance/README.md

```diff
@@ -151,17 +178,26 @@ Usage: `mvn verify -Ptest[,cluster] [-DtestParameter=value]`.
 | `userThinkTime` | Pause between individual scenario steps. | `5` |
 | `refreshTokenPeriod`| Period after which token should be refreshed. | `10` |
 
-#### Addtional Parameters of `keycloak.BasicOIDCSimulation`
+#### Parameters specific to `keycloak.BasicOIDCSimulation`
 
 | Parameter | Description | Default Value |
 | --- | --- | --- | 
 | `badLoginAttempts` | | `0`  |
 | `refreshTokenCount` | | `0` |
 
+#### Examples:
+
+- Run test with default test and dataset parameters:
+
+`mvn verify -P test `
+
+- Run test specific test and dataset parameters:
+
+`mvn verify -P test -Dtest.parameters=basic-oidc -Ddataset=100u2c`
 
-Example:
+- Run test with specific test and dataset parameters, overriding some from command line:
 
-`mvn verify -Ptest -Dgatling.simulationClass=keycloak.AdminConsoleSimulation -Ddataset=100u -DrunUsers=1 -DmeasurementPeriod=60 -DuserThinkTime=0 -DrefreshTokenPeriod=15`
+`mvn verify -P test -Dtest.parameters=admin-console -Ddataset=100u2c -DrampUpPeriod=30 -DwarmUpPeriod=60 -DrunUsers=10`
```

Comment:
Didn't we remove -DrunUsers?

Reply:
Not yet, KEYCLOAK-6512 is not merged yet. We'll have to rebase.